js/column.js

```diff
@@ -270,6 +270,18 @@ ReferenceColumn.prototype = {
         return this._comment;
     },
 
+    get commentDisplay() {
+        if (this._comment === undefined) {
+            if (this.sourceObject && this.sourceObject.comment && this.sourceObject.comment_display) {
+                // only change commentDisplay if comment and comment_display are both defined
+                this._commentDisplay = this.sourceObject.comment_display;
```

Comment:
This comment is applicable to all the different places that you're populating the `commentDisplay` API value from the annotation. If you're not doing any extra check for "valid" string values (`tooltip`, `inline`) in ermrestjs, then in your chaise code you have to make sure you're properly handling this.  I think an invalid value, should be treated as if the annotation is missing and therefore should be treated the same as `tooltip`.